WinUIGallery/MainWindow.xaml.cs

```diff
@@ -42,10 +42,24 @@ public MainWindow()
 
     private void RootGrid_Loaded(object sender, RoutedEventArgs e)
     {
+        // We need to set the minimum size here because the XamlRoot is not available in the constructor.
+        WindowHelper.SetWindowMinSize(this, 640, 500);
+
+        if (sender is FrameworkElement rootGrid && rootGrid.XamlRoot is not null)
+        {
+            rootGrid.XamlRoot.Changed -= RootGridXamlRoot_Changed;
```

Comment:
Would there be a case where RootGrid.Loaded gets raised twice and this would avoid a duplication of event subscriptions?